cockroachcloud/export-metrics.md

```diff
@@ -233,6 +235,10 @@ Where:
 
 To update an existing {{ site.data.products.dedicated }} metrics export configuration, make any necessary changes to your cloud provider configuration (e.g., AWS CloudWatch or Datadog), then issue the same `POST` Cloud API command as shown in the [Enable metrics export](#enable-metrics-export) instructions for your cloud provider with the desired updated configuration. Follow the [Monitor the status of a metrics export configuration](#monitor-the-status-of-a-metrics-export-configuration) instructions to ensure the update completes successfully.
 
+## Disable Cloud Console's local storage of metrics (optional)
+
+If you rely on external tools such as AWS CloudWatch or Datadog for storing and visualizing your cluster's time-series metrics, Cockroach Labs recommends that you disable the Cloud Console's storage of time-series metrics by setting the [`timeseries.storage_enabled` cluster setting](/docs/v23.1/cluster-settings.html#setting-timeseries-storage-enabled) to `false`. When this setting is `false`, Cloud Console does not store time-series cluster metrics, but still exposes them via the [`metricexport` endpoint](#metricexport-endpoint). The Cloud Console pages that previously showed data about these metrics are still available, but visualizations are blank because the Cloud Console no longer persists these metrics.
```

Comment:
> The Cloud Console pages that previously showed data about these metrics are still available but visualizations are blank because

I.e. the pages will exist but there will be no data there? If so, could this be slightly clarified?

> no longer persists these metrics

This use of 'persists' seems like dev jargon so I'd propose 'maintains', if accurate.

Reply:
@mdlinville Just realized that I suggested "maintains" above when I should have suggested "retains"—FWIW! Edited my comment.